style.css

```diff
@@ -20,6 +20,13 @@ html {
     margin: 0;
 }
 
+.sub-contaner {
```

Comment:
re -arrange the order of this css such that we have all html raw components at the top, then pure classes